builder/src/firmware.rs

```diff
@@ -243,16 +254,31 @@ pub mod driver_tests {
         ..BASE_FWID
     };
 
+    pub const TEST_LMS_24_HW_LATEST: FwId = FwId {
+        bin_name: "test_lms_24_hw_latest",
+        ..HW_LATEST_FWID
+    };
+
     pub const TEST_LMS_32: FwId = FwId {
         bin_name: "test_lms_32",
         ..BASE_FWID
     };
 
+    pub const TEST_LMS_32_HW_LATEST: FwId = FwId {
+        bin_name: "test_lms_32_hw_latest",
+        ..HW_LATEST_FWID
+    };
+
     pub const TEST_NEGATIVE_LMS: FwId = FwId {
         bin_name: "test_negative_lms",
         ..BASE_FWID
     };
 
+    pub const TEST_NEGATIVE_LMS_HW_LATEST: FwId = FwId {
+        bin_name: "test_negative_lms_hw_latest",
+        ..BASE_FWID
```

Comment:
Shouldn't this be HW_LATEST_FWID?

Reply:
Done -: https://github.com/chipsalliance/caliptra-sw/pull/1359/commits/e270e7434bafd7d3fa4ae24e0fe757d271096bb4